src/org/labkey/test/LabKeySiteWrapper.java

```diff
@@ -1007,6 +1007,12 @@ public PipelineStatusTable goToDataPipeline()
         return new PipelineStatusTable(this);
     }
 
+    public void goToExternalToolPage()
+    {
+        clickUserMenuItem("External Tool Settings");
+        waitForText("API keys are used to authorize");
```

Comment:
I believe this will fail if API keys aren't enabled on the server. Likely not important for current test coverage